config/workflows/accessionWF.xml

```diff
@@ -27,6 +27,10 @@
     <label>Update DOI Metadata</label>
     <prereq>publish</prereq>
   </process>
+  <process batch-limit="100" error-limit="10" name="update-orcid-work">
+    <label>Update ORCID work</label>
+    <prereq>publish</prereq>
+  </process>
```

Comment:
@justinlittman I see this process has been set up similarly to `update-doi` in terms of the configured limits and the prerequisite which makes me wonder if `update-orcid-work` should also be a prerequisite of `end-accession` below (as is `update-doi`).

Reply:
It shouldn't matter, but I added anyway.